core/src/navigation/LeftNav.html

```diff
@@ -158,6 +158,17 @@
         this.fire('handleClick', { node });
       },
 
+      handleKey(event) {
+        let code = event.code;
```

Comment:
Works as expected. Just a couple of cosmetic comments:
- All variables can be defined with const
- Maybe the `win` variable initialization would be simpler with `iframe.contentWindow || (iframe.contentDocument && iframe.contentDocument.window))`
- Actually do we really need `|| (iframe.contentDocument ? iframe.contentDocument.window : null)`? contentDocument is supposed to be like document and document has no focus method.